apps/api/v2/src/modules/organizations/inputs/create-organization-membership.input.ts

```diff
@@ -0,0 +1,22 @@
+import { IsBoolean, IsOptional, IsString, IsInt } from "class-validator";
+
+import { MembershipRole } from "@calcom/prisma/enums";
+
+export class CreateOrgMembershipDto {
+  @IsInt()
+  readonly teamId!: number;
+
+  @IsInt()
+  readonly userId!: number;
+
+  @IsOptional()
+  @IsBoolean()
+  readonly accepted?: boolean = false;
```

Comment:
Following up on [campsite comment](https://app.campsite.co/cal/posts/rzfttn7s2cm0#comment-oupbuqm9zi53)  - my first instint is that we should not allow passing accepted at all, because I think the users should manually accept the membership.

Reply:
Let's discuss in campsite further on the approach here 🙏

Reply:
I think when an admin is creating a membership directly he should be able to do whatever

for the /invite endpoint it's a different story and accepted should not be passed